### ISSUE TITLE
Add 60 FPS limit

### ISSUE THREAD

**Issue by r00ster91** [teeworlds/teeworlds#1537]
Please add an 60 FPS limit as an option. (Or is there an FPS limit? I can't find it.)

My computer crashes and gets pretty hot after some time when there's a way too uselessy high framerate. I'm sure I'm not the only one with this problem. And also without an FPS limit, more battery power is getting wasted for pretty much nothing. Mostly you just can't see more than 60-80 FPS. I don't think an optional FPS limit would harm.

**Comment by Sonix-** [comment]
If v-sync doesn't help, then yeah an option to limit fps should be added. A slider to set your own limit would be ideal.

**Comment by oy** [comment]
Enabling vsync should limit it to the framerate your monitor provides, not necessary to 60FPS, but still some limit. Are you using the 0.7 version?

**Comment by r00ster91** [comment]
Yes I'm using 0.7.0 and VSync just limits it to 250 FPS which is way too much.

**Comment by oy** [comment]
What OS are you on? What graphics card do you have?

**Comment by r00ster91** [comment]
Operating system: Microsoft Windows Version 10.0.17134.345
Graphics card: NVIDIA GeForce 920M

**Comment by Sonix-** [comment]
I would still suggest adding a FPS limiter, because there are many people, myself included, that play without v-sync for various reasons
